packages/next/src/utils/create-cli-options.ts

```diff
@@ -3,8 +3,7 @@ export function createCliOptions(
 ): string[] {
   return Object.entries(obj).reduce((arr, [key, value]) => {
     if (value !== undefined) {
-      const kebabCase = key.replace(/[A-Z]/g, (m) => '-' + m.toLowerCase());
-      arr.push(`--${kebabCase}=${value}`);
+      arr.push(`--${key}=${value}`);
```

Comment:
I don't think we can remove this as it would break our `build` executor when options are passed

Reply:
sure, closing out this PR
probably fine to just add an exception for keepAliveTimeout since it's the only camelCased flag